### PR TITLE
#5139 In any case can enter the setting page to use restart workspace

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5139 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js [hunk 1]
```diff
@@ -67,14 +71,40 @@ const finalize$ = ({ zookeeperKey, brokerKey, workerKey, workspaceKey }) =>
...
+  const isNonExistedObject = defer(() => api)
+    .pipe(
+      map((res) => res.data),
+      map((dataArray) =>
+        dataArray.find(
+          (data) =>
+            data.name === objectKey.name && data.group === objectKey.group,
+        ),
+      ),
+      map((object) => isEmpty(object)),
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:94]: If we want the return value to be `observable(true|false)`, maybe it's time to introduce [every](https://rxjs-cn.github.io/learn-rxjs-operators/operators/conditional/every.html) :)

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**